.github/workflows/sanitize.yml

```
@@ -61,24 +61,19 @@
       with:
         path: ./bin/linux-x64-debug-asan/libcsv
         key: libcsv-sanitizer-llvm-${{ hashFiles('./deps/libcsv/ChangeLog') }}
-
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)